app/src/main/res/layout/activity_main.xml

```diff
@@ -66,6 +66,31 @@
         app:layout_constraintTop_toBottomOf="@id/btn_set_notification_time" />
 
 
+    <RadioGroup
+        android:id="@+id/rg_single_repeat"
+        app:layout_constraintTop_toBottomOf="@id/tv_alarm_date"
+
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+
+        <RadioButton
+            android:id="@+id/rb_single_alarm"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+            android:text="@string/single_alarm"
```

Comment:
This requires a change to strings.xml, too, right?

Reply:
Yes, DAMMIT (not at you). Android Studio does so much stuff, I really thought it would keep track of what files should be pushed.